style-packages.md

```diff
@@ -0,0 +1,255 @@
++++
+date        = "2017-01-14T11:27:27-04:00"
+title       = "Style guideline for Go packages"
+description = "Best practices to organize and name Go packages"
+tags        = [ "readability" ]
+type        = "archived"
+draft       = false
++++
+
+Go is about naming and organization as much as everything else in the language.
+Go code lives in a package, a package is the entry point to access Go code. Understanding
+and establishing good practices around packages is important to author effective Go code.
+
+----
+
+## Organization
+
+Let's begin with suggestions how you should organize Go code and explain conventions about
+locating Go packages. Well-organized Go code is easy to discover,
+use and read. Well-organized code is as critical as well designed APIs. The location, name,
+and the structure of your packages are the first elements your users see and interact with.
+
+### Organize by responsbility
+
+A common baggage from other languages is to organize types together in a package
+called models or types. In Go, we organize code by their functional responsbilities.
```

Comment:
I moved this after the "keep types closer" section. I will give more justifications.

Done